examples/plotting/file/airports_map.py

```diff
@@ -1,8 +1,9 @@
 '''This example shows the same data on two separate tile plots.
 The left plot uses the built-in CartoDB tile source, and the right plot uses
-a customized tile source configured for OpenStreetMap
+a customized tile source configured for OpenStreetMap.
 
 .. bokeh-example-metadata::
+    :sampledata: airport
```

Comment:
Does this add also a link from [sampledata](`https://docs.bokeh.org/en/latest/docs/reference/sampledata.html?highlight=sampledata#module-bokeh.sampledata.airports`) to the example or is this working only in one direction? If this is one way, I would suggest to add also linkes from this page to relateed examples, if possible. I think this will improve the user experience.

Reply:
@mosc9575 currently one way. I have had the same thought about automatically cross-linking examples for several years, it really just comes down to someone having the time to work on developing our sphinx extensions to support that.

Reply:
Anything a "Newbe" can do about this? I would love to participate.

Reply:
Certainly possible, but it would be a non-trivial time investment (not trying to discourage, just being up front) especially without prior Sphinx extension experience. You can look over all of our existing sphinx extensions here:

https://github.com/bokeh/bokeh/tree/branch-3.0/bokeh/sphinxext

We used to have a custom parser for `.py` files in the repo but moved away from that at one point. I think we would probably need look at adding that back to process all `.py` files automatically to: 

* generate standalone pages for each with metadata renderered
* add them to an index somehow
* look for places to link them besides the index (e.g. appropriate places in user/ref guide)

Reply:
I want to give this a try. Does a ticket exists I can link a (possible) PR to? If not I will open a Feature Request.

Reply:
@mosc9575 I don't think we have an issue about that yet. Feel free to [create an issue](https://docs.bokeh.org/en/latest/docs/dev_guide/creating_issues.html), and also a PR! Other than looking at our existing Sphinx extensions, the [Sphinx documentation itself](https://www.sphinx-doc.org/en/master/development/index.html) might be a good starting point. Thank you! 😀